### PR TITLE
Add a bio for @horacechan

### PR DESCRIPTION
**Pull request by horacechan**
This Pull request adds a bio for @horacechan as request in #21.

@TrevorTheIntuitCoder What do you think? :star: or :hankey:

### CODE REVIEW

**Comment by TrevorTheIntuitCoder** [comment]
:hankey: x :100:

File: docs/horacechan.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Horace bio
+
+- **Name:** Horace Chan
```
Comment (TrevorTheIntuitCoder) [inline docs/horacechan.md]: what is your middle name?

Comment (horacechan) [inline docs/horacechan.md]: I don't have one. :smile:

**Comment by TrevorTheIntuitCoder** [comment]
:shipit:

**Pull request merged**